src/EDA_Census.py

```diff
@@ -13,15 +13,29 @@
 def main():
 
     input_file = args.input_file
-    ouput_file = args.ouput_file
```

Comment:
we have another 'output_file' defined for arg on line 16.
we can remove one of them. This line has a typo on the 'output_file'('ouput_file'